### DISCUSSION TITLE
Add Virtual fields for computing

### DISCUSSION THREAD

**Discussion by halwesit** [directus/directus#15614] [Archive]
Hello Everyone.

If you have used the KystoneJS, it has a custom field type known as Virtual field, It can be helpful to have read-only fields computed on the fly when you query them. 
see it in their docs: [Virtual Fields](https://keystonejs.com/docs/guides/virtual-fields#virtual-fields)

I need this type of field to compute other fields (it's better if we could access other collections) and save the calculated data into the virtual field.

Thanks.

**Comment by azrikahar** [comment]
Should be the same use case as #4231 and #3079. Feel free to upvote and keep this "computed/virtual field" topic in those discussions 👍
